libs/application/templates/parental-leave/src/fields/InReviewSteps/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC } from 'react'
+import { useLocale } from '@island.is/localization'
+
+import { FieldBaseProps } from '@island.is/application/core'
+import { Box, Button, Text } from '@island.is/island-ui/core'
+
+import { mm } from '../../lib/messages'
+import ReviewSection, { reviewSectionState } from './ReviewSection'
+
+type stateMapEntry = { [key: string]: reviewSectionState }
+type statesMap = {
+  otherParent: stateMapEntry
+  employer: stateMapEntry
+  vinnumalastofnun: stateMapEntry
+}
+const statesMap: statesMap = {
+  otherParent: {
+    otherParentApproval: 'In progress',
+    otherParentRequiresAction: 'Requires action',
+    employerApproval: 'Complete',
+    vinnumalastofnunApproval: 'Complete',
+  },
+  employer: {
+    employerApproval: 'In progress',
+    employerRequiresAction: 'Requires action',
+    vinnumalastofnunApproval: 'Complete',
+  },
+  vinnumalastofnun: {
+    vinnumalastofnunApproval: 'In progress',
+    vinnumalastofnunRequiresAction: 'Requires action',
+    approved: 'Complete',
+  },
+}
+
+const InReviewSteps: FC<FieldBaseProps> = ({ application }) => {
+  const { formatMessage } = useLocale()
+
+  return (
+    <Box marginBottom={10}>
+      <Box>
+        <Text as="div">
+          {formatMessage(mm.reviewScreen.desc)}{' '}
+          <Box display="inlineBlock" marginLeft={1} marginRight={2}>
+            <Button
+              colorScheme="default"
+              iconType="filled"
+              // TODO: Add onClick in next PR
+              size="small"
+              type="button"
+              variant="text"
+            >
+              {formatMessage(mm.reviewScreen.buttonsView)}
+            </Button>
+          </Box>
+          <Box display="inlineBlock">
+            <Button
+              colorScheme="default"
+              iconType="filled"
+              // TODO: Add onClick in next PR
+              size="small"
+              type="button"
+              variant="text"
+            >
+              {formatMessage(mm.reviewScreen.buttonsEdit)}
```

Comment:
this one is a bit more tricky to implement. You need to use the submitApplication graphql function

Reply:
Ooh, sounds good, I'll look into this one next.